### ISSUE TITLE
Docs need improvement [suggestion]

### ISSUE THREAD

**Issue by falafflepotatoe** [Rich-Harris/ramjet#39]
Small rant sorry...

You should mention somewhere that elements have to be hidden first then do `ramjet.transform()`, and use `{ done: => { ramjet.show(target) }}`... and also maybe that it doesnt like jquery elements.

If the docs do say that, my apologies, I couldn't find it

Reason is all your demos use hidden elements, but its not clear how to interact with a hidden element, and the effect doesnt work at all with hidden elements..

I originally tried to setup a fiddle with jquery and ramjet and it didnt work at all, then I tried to use a hidden element as a target transition and it didnt work, so I tried all the variations of methods to hide an element, display:none, visibility:hidden, jquery hide, etc.

anyway after 30 minutes i got this: https://jsfiddle.net/davwcwmr/ cool!

**Comment by weislanes** [comment]
Yes, that would be very helpful as spent a similar amount of time figuring this out. Thanks for the jsfiddle.
